cmd/kubeadm/app/constants/constants.go

```diff
@@ -499,8 +499,9 @@ var (
 		20: "3.4.13-0",
 		21: "3.4.13-0",
 		22: "3.5.3-0",
-		23: "3.5.3-0",
-		24: "3.5.3-0",
+		23: "3.5.4-0",
+		24: "3.5.4-0",
```

Comment:
Please change all 3.5.x versions in kubeadm to 3.5.4

Reply:
Done.